plugins/automation/spec/system/page_objects/discourse_automation/automation.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class Automation < PageObjects::Pages::Base
+      def visit(automation)
+        super("/admin/plugins/discourse-automation/#{automation.id}")
+        self
+      end
+
+      def set_name(name)
+        form.find('input[name="automation-name"]').set("aaaaa")
+        self
+      end
+
+      def has_error?(message)
+        form.has_content?(message)
+      end
+
+      def has_name?(name)
+        form.find_field("automation-name").value == name
```

Comment:
```suggestion
        form.find_field("automation-name", with: name)
```

Small one but this will ensure that capybara waits for a field with a given value instead of just waiting for the field to be present.

Reply:
ah yes right thanks